task/sast-snyk-check/0.1/sast-snyk-check.yaml

```diff
@@ -55,7 +55,8 @@ spec:
           SNYK_TOKEN="$(cat ${SNYK_TOKEN_PATH})"
           export SNYK_TOKEN
         else
-          note="Task $(context.task.name) skipped: If you wish to use the Snyk code SAST task, please create a secret named snyk-secret with the key "snyk_token" containing the Snyk token."
+          to_enable_snyk=[here](enable-snyk-check-for-a-product.md)
+          note="Task $(context.task.name) skipped: If you wish to use the Snyk code SAST task, please create a secret named snyk-secret with the key "snyk_token" containing the Snyk token by following the steps given ${to_enable_snyk}"
```

Comment:
This won't work, note doesn't parse markdown format (also any relative links won't work here), you can only use absolute raw URL to github

Reply:
probably should add this doc in the [user docs](https://redhat-appstudio.github.io/docs.appstudio.io/Documentation/main/how-to-guides/testing_applications/con_test-overview/#product-security-tests) first and add that link here.

Reply:
+1 to link the user guide doc, then we can only maintain one doc.

Reply:
done